internal/pkg/cli/deploy/workload.go

```diff
@@ -147,6 +149,7 @@ type workloadDeployer struct {
 	mft           interface{}
 	rawMft        []byte
 	workspacePath string
+	uri           string
```

Comment:
does this need to be on the `workloadDeployer`? it looks like it's only used in the `uploadContainerImages()` function 🤔

Reply:
I can modify function `Login`  to return the uri as well 🤔

Reply:
removed `uri` from `workloadDeployer`. Instead modified the `login` to return uri as well.